frontend/cypress/integration/featureFiles/istio_config_type_filters.feature

```diff
@@ -0,0 +1,95 @@
+@istio-page
+Feature: Kiali Istio Config page
+
+  On the Istio Config page, an admin should be able to choose all of the filters present. 
+  The admin should also be able close a single filter either with it's cross, or with the Close All feature.
+
+  Background:
+    Given user is at administrator perspective
+    And user is at the "istio" page
+    And user selects the "istio-system" namespace
+    And user filters by "Istio Type" 
+    And no filters are active
+
+    Scenario Outline: All of the filters should be available for turning on
```

Comment:
This test works but I feel the run time is maybe not worth the test.  In essence it tests that the menu has all of the expected options, but it does it one at a time, relogging in, reselecting the Istio Config menu, and then selecting the filter.  I feel like all of the the options could be tested at the same time.   We are not really trying to test patternfly menus, that's for PF to do, we're trying to test Kiali.  As such, I think we would be better just making sure that all of the expected options are there and then activate a filter (or two) check that the filtering actually works, and that deactivation works.

The same comment applies for the tests below that are repeating the same thing for every menu option.  This test is the second longest running cypress feature file, and the two new feature files combined take over 3 minutes, which is over 10% of the total runtime.

  │ ✔  featureFiles/istio_config_type_filt      02:13       39       39        -        -        - │
  │    ers.feature                                                                                 │
  
  │ ✔  featureFiles/istio_config_validatio      00:55       17       17        -        -        - │
  │    n_filters.feature

Reply:
Decided to rework this entirely. Closing the PR for now. I am going to make a new one once it's done.